docs/source/gcp/examples/pubsub-topics.rst

```diff
@@ -0,0 +1,27 @@
+Pub/Sub - Find Topics which Violate Naming Convention
+=======================================================
```

Comment:
please leave this example out, naming considerations on resources for an org don't need separate per resource examples.

Reply:
Done.